experimental/dds/tree2/src/shared-tree-core/branch.ts

```diff
@@ -184,9 +189,12 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 			change,
 		});
 
+		const taggedChange = tagChange(change, revision);
 		const changeEvent = {
 			type: "append",
-			change: tagChange(change, revision),
+			get change() {
```

Comment:
This does not need to be a getter, right?
Same for all in this file, except for the one in `rebaseOnto`.

Reply:
🤦‍♂️